FreeRTOS/FreeRTOS
# Update MPU Demos for Kernel V10.6.0

Description
-----------
This PR ads Macros in Demo config file, for compatibility with FreeRTOS Kernel V10.6.0

Test Steps
-----------
Run soak tests for ARMv7 and ARMv8M .

Checklist:
----------




By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
